COMMUNITY-PROJECTS.md

```diff
@@ -28,3 +28,9 @@ Ongoing
 
 #### ipfsUploader
 Ongoing
```

Comment:
Would you mind if we linked to your repo here? > https://github.com/rej156/ipfs-uploader <